Wikipedia/Code/ExploreViewController.swift

```diff
@@ -192,11 +200,20 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
         return titleView
     }()
 
-    @objc func userDidTapSettings() {
+    @objc func userDidTapProfile() {
         DonateFunnel.shared.logSettingsDidTapSettingsIcon()
      
-        settingsPresentationDelegate?.userDidTapSettings(from: self)
+        // settingsPresentationDelegate?.userDidTapProfile(from: self)
+        let profileView = WMFProfileView(isLoggedIn: true)
+        let hostingController = UIHostingController(rootView: profileView)
+        hostingController.modalPresentationStyle = .pageSheet
+
+        if let sheetPresentationController = hostingController.sheetPresentationController {
+            sheetPresentationController.detents = [.large()]
+            sheetPresentationController.prefersGrabberVisible = true
+        }
 
+        present(hostingController, animated: true, completion: nil)
```

Comment:
@l-olson1214 One more thing - I notice WMFProfileViewController isn't used yet. You could replace these lines with this to use it:

```
let profileViewController = WMFProfileViewController()
present(profileViewController, animated: true, completion: nil)
```